cmd/buildkitd/main_containerd_worker.go

```diff
@@ -46,6 +48,14 @@ func init() {
 		defaultConf.Workers.Containerd.Namespace = defaultContainerdNamespace
 	}
 
+	if defaultConf.Workers.Containerd.Runtime == "" {
+		if runtime.GOOS == "freebsd" {
+			defaultConf.Workers.Containerd.Runtime = "wtf.sbk.runj.v1"
```

Comment:
Maybe add a TODO saying it should be removed once https://github.com/containerd/containerd/pull/8964 comes with a containerd release?